app/models/campus_solutions/student_resources.rb

```diff
@@ -116,6 +117,8 @@ def filter_role_based_link_settings(link_settings)
           true unless roles[:concurrentEnrollmentStudent]
         when :grad_change_of_academic_plan_add
           true unless roles[:graduate]
+        when :dissertation_signature, :expected_grad_term_add
+          true unless (roles[:graduate] || roles[:law])
```

Comment:
@nkutub - I'm concerned about what this returns if the unless statement applies.  It returns nil... does that still bring the desired behavior?

Would this pattern work?
```ruby
when :dissertation_signature, :expected_grad_term_add
  !(roles[:graduate] || roles[:law])
```

Reply:
@redconfetti  nil returns as false in ruby https://www.ruby-lang.org/en/documentation/faq/6/ . It is working with previous case statements and see no need to add potential bugs for this effort